modules/ROOT/pages/add-api-alert-task.adoc

```diff
@@ -1,21 +1,27 @@
-= Adding an API Alert
+= Add API Alerts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-. In API Manager 2.x, in API Administration, select an API instance, and on the API instance dashboard, select Alerts.
-. Click Add Alert and set the alert options:
-* Name: Type an alert name, for example `red-alert`.
-* Enabled: Accept the default, Enabled, for example.
-* Severity: Accept the default, Critical.
-* Alert type: Select Request Count, for example.
-* When number of occurrences are: Select Greater.
-* than: Select 1.
-* For at least: Select an entry in the range 1 - 999999999 to specify the time intervals as a sliding windows.
-* Recipients: Select the names of users in your business group, or type the email addresses, of users who will receive notification when the alert fires. Press RETURN.
+You can add one or more API alerts for your API instances to track and manage APIs.
+
+To add an API alert:
+
+. Navigate to *API Manager* > *API Administration* and click the API instance for which you want to add an alert.
+. From the left navigation select *Alerts* > *Add alert*.
+. Specify the alert options:
+* *Name*: Type an alert name, for example `trial-alert`.
+* *Enabled*: Accept the default option: *Enabled*.
 +
-. Create the alert.
+If you no longer need the alert, you can disable it any time later.
+* *Severity*: Select a severity for the alert, for example, *Info*.
+* *Alert type*: Select the type of alert to create, for example, *Request Count*:
+** *When number of occurrences are*: Select *Greater*.
+** *than*: Type 1.
+** *For at least*: Type a value in the range 1 - 999999999 to specify the time intervals as a https://www.techopedia.com/definition/869/sliding-window[sliding windows].
+* *Recipients*: Select the names of the users in your business group, or type the email addresses of users who must receive the notifications when the alert is generated, and press return.
```

Comment:
Suggest omitting, per above.
If kept, these require reformatting in our bullet style.
Also, "press return"? What is the actual user action? "pressing return" is not a UI action but a keyboard action.

Reply:
Removed! Done! Thanks!